greengrass_parsec_workshop/build_demo.sh

```diff
@@ -15,7 +15,12 @@ function update_git() {
   # Need to update the protobuf from parsec
   git submodule init
   git submodule update
-
+  pushd ./aws-greengrass-parsec-provider/
+  git submodule init
+  git submodule update
+  pushd ./aws-greengrass-parsec-provider/parsec-client-java/
```

Comment:
Firstly, this is an incorrect path. after the first pushd the current directory is `aws-greengrass-parsec-provider` already.
Secondly, we don't need to run init for all the submodules one by one. You can simply do:
```
  git submodule update --init --recursive
```

Reply:
Thanks.. this is fixed.. we had it verbose step by step from previous debugging.. but cleaned up now, thanks @anta5010